src/common/servlist.c

```diff
@@ -119,6 +119,11 @@ static const struct defaultserver def[] =
 
 	{"DarkMyst", 0, 0, 0, LOGIN_SASL, 0, TRUE},
 	{0,			"irc.darkmyst.org"},
+	
+	{"darkscience", 0, 0, 0, LOGIN_SASL, 0, TRUE},
```

Comment:
Wrap in `#ifdef USE_OPENSSL` like some of the other networks.

Fix whitespace to match others.

Reply:
Thanks for the feedback, I am not sure how the whitespace got messed up (I was editing using github web).

I have made the changes you suggested, the entry will be completely removed if openssl is not available, as the network does not allow plaintext. I hope that is ok.

Reply:
You are mixing spaces when the file uses tabs.

Reply:
File seems to use a mixture of tabs and spaces here:

![image](https://user-images.githubusercontent.com/667455/83339579-a5188900-a2cf-11ea-91a4-72455e562e7b.png)

I have changed the style to conform to those above.

Reply:
wait, nevermind, I must have broken the file myself somehow. I will fix.

Reply:
You reformatted the entire file...

Reply:
Yeah, I saw. Fixed immediately in https://github.com/hexchat/hexchat/commit/46fd9a460508f019b2be13edd616ef91432bd66a